net/openssh/Portfile

```diff
@@ -5,8 +5,7 @@ PortSystem          1.0
 PortGroup           compiler_blacklist_versions 1.0
 
 name                openssh
-version             9.7p1
-revision            1
+version             9.8p1
```

Comment:
```suggestion
version             9.8p1
revision            0
```

Reply:
Yes, but that was changed some years ago.